torch/testing/_internal/common_methods_invocations.py

```diff
@@ -15070,7 +15070,7 @@ def reference_flatten(input, start_dim=0, end_dim=-1):
     OpInfo('lerp',
            dtypes=floating_and_complex_types_and(torch.bfloat16, torch.half),
            dtypesIfCUDA=floating_and_complex_types_and(torch.chalf, torch.half, torch.bfloat16),
-           dtypesIfROCM=floating_and_complex_types_and(torch.half, torch.bfloat16),
+           dtypesIfROCM=floating_and_complex_types_and(torch.chalf, torch.half, torch.bfloat16),
```

Comment:
@alugorey This should be fine, although IIUC, since the `dtypesIfCUDA` value is identical to `dtypesIfROCM` in this case, you could even get rid of this line, and it will just take the `dtypesIfCUDA` value. @jeffdaily, can you please confirm?

Reply:
Correct, `dtypesIfROCM` defaults to `dtypesIfCUDA` if not explicitly set.

Reply:
ah, didn't know that was a feature. Okay, i'll remove